planet/src/main/webapp/js/itinerary.js

```diff
@@ -27,6 +27,8 @@ window.addEvent = addEvent;
 window.saveEvents = saveEvents;
 window.generateItinerary = generateItinerary;
 
+const database = firebase.database();
```

Comment:
Why do we have to initialize the database more than once? it is also a global variable

Reply:
i didn't have them before but one day it stopped working and when I added them it works again...